.gitignore

```diff
@@ -30,3 +30,5 @@
 /yarn-error.log
 
 spec/reports/pacts
+
+/app/assets/javascripts/templates.js
```

Comment:
Looks like you're missing a new line character on this file.

Reply:
I've added a new line here but for some reason it's not showing in files changed. It is there though I promise!